apps/contacts/resource.py

```diff
@@ -44,7 +45,8 @@ class ContactsResource(Resource):
         },
         'last_name': {
             'type': 'string',
-            'required': False
+            'required': False,
+            'mapping': not_analyzed
```

Comment:
not sure this is correct, why not analyzed?

Reply:
changed to `{'type': 'string'}`